src/client/components/pages/Courses/CourseInstanceTable.tsx

```diff
@@ -200,51 +385,57 @@ const CourseInstanceTable: FunctionComponent<CourseInstanceTableProps> = ({
         )}
       </TableHead>
       <TableBody>
-        {courseList.map(
+        {filteredCourses(courseList.map(
```

Comment:
As noted above, I think this filtering should be done in the `CoursesPage` component, instead of inside the `CourseInstanceTable`. That might bring back the error you were seeing about hooks being called a different number of times, so we may need to look into that a bit more...